samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/pdf/HTMLWorker.java

```diff
@@ -176,6 +179,23 @@ else if (src.startsWith("temp://")) {
 					log.error(e.getMessage(), e);
 				}
 			}
+			else if (src.startsWith("data:image/")) {
+				Image img = null;
+				final String base64Data = src.substring(src.indexOf(",") + 1);
+				try {
+					img = Image.getInstance(Base64.decode(base64Data));
+				} catch (Exception e) {
+					log.error(e.getMessage(), e);
```

Comment:
The log statement here should be more descriptive, and should it be a warning vs an error?

Reply:
Just changed!